components/cli/pkg/commands/instance/logs_test.go

```diff
@@ -27,41 +27,62 @@ import (
 )
 
 func TestRunLogs(t *testing.T) {
-	logsMap := make(map[string]string)
-	componentLogsMap := make(map[string]string)
-	logsMap["employee"] = "employee cell is running"
-	componentLogsMap["employee:job"] = "job component is running"
-	mockKubeCli := test.NewMockKubeCli(test.WithCellLogs(logsMap), test.WithComponentLogs(componentLogsMap))
+	mockKubeCli := test.NewMockKubeCli(test.WithRunningInstance(true))
```

Comment:
As mentioned before, it is better if instance availability was not hardcoded using "WithRunningInstance".

Reply:
Done